src/RequestParser.php

```diff
@@ -133,7 +133,7 @@ public function parseBody($content)
                 return;
             }
 
-            if (strtolower($headers['Content-Type']) == 'application/x-www-form-urlencoded') {
+            elseif (strpos(strtolower($headers['Content-Type']), 'application/x-www-form-urlencoded') === 0) {
```

Comment:
Either way is fine, and I can change it if you want.  

I can't find anywhere in the [the official documents](http://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html) that specifies that the header name *must* appear as the first character on a line, although I've never seen white space before a header in practice.

Perhaps this may be the least ambiguous:

```php
strpos(strtolower(trim($headers['Content-Type'])), 'application/x-www-form-urlencoded') === 0
````